docs/layouts/shortcodes/generated/cluster_configuration.html

```diff
@@ -80,5 +80,23 @@
             <td>Boolean</td>
             <td>Whether to convert all PIPELINE edges to BLOCKING when apply fine-grained resource management in batch jobs.</td>
         </tr>
+        <tr>
+            <td><h5>process.jobmanager.working-dir</h5></td>
+            <td style="word-wrap: break-word;">process.working-dir</td>
+            <td>String</td>
+            <td>Working directory for the JobManager process. The working directory can be used to store information that can be used upon process recovery. If the not configured, then it will default to <code class="highlighter-rouge">process.working-dir</code>.</td>
+        </tr>
+        <tr>
+            <td><h5>process.taskmanager.working-dir</h5></td>
+            <td style="word-wrap: break-word;">process.working-dir</td>
+            <td>String</td>
+            <td>Working directory for the TaskManager process. The working directory can be used to store information that can be used upon process recovery. If the not configured, then it will default to <code class="highlighter-rouge">process.working-dir</code>.</td>
+        </tr>
+        <tr>
+            <td><h5>process.working-dir</h5></td>
+            <td style="word-wrap: break-word;">io.tmp.dirs</td>
+            <td>String</td>
+            <td>Working directory for Flink processes. The working directory can be used to store information that can be used upon process recovery. If the not configured, then it will default to <code class="highlighter-rouge">io.tmp.dirs</code>.</td>
```

Comment:
This is mapping multiple directories into a single directory. Can you add an explanation on which directory would be picked in case of multiple io dirs?

Reply:
Yes, will add it in the corresponding PR for this change.